src/Graph1/Graph1.tsx

```diff
@@ -1,32 +1,34 @@
 import { ProgressBar } from "../components/ProgressBar";
-
+import { useGraphsContext } from "@features/themes";
 const GRAPH_1_DATA = [
-  {
-    label: "etc",
-    value: 30,
-  },
-  {
-    label: "elit",
-    value: 60,
-  },
-  {
-    label: "ess",
-    value: 80,
-  },
+	{
+		label: "etc",
+		value: 30,
+	},
+	{
+		label: "elit",
+		value: 60,
+	},
+	{
+		label: "ess",
+		value: 80,
+	},
 ];
 
-// TODO: USE GRID INSTEAD OF FLEX 
+// TODO: USE GRID INSTEAD OF FLEX
 export function Graph1() {
-  return (
-    <div className="flex flex-col h-full gap-1">
-      {GRAPH_1_DATA.map(({ label, value }, index) => (
-        <ProgressBar
-          key={label}
-          label={label}
-          value={value}
-          colorVariant={index}
-        />
-      ))}
-    </div>
-  );
+	const { graphStates } = useGraphsContext();
+
+	return graphStates["Graph 1"] ? (
```

Comment:
```
export function Graph1() {
  const { graphStates } = useGraphsContext();

  if (!graphStates["Graph 1"]) return null;

  return (
    <div className="flex flex-col h-full gap-1">
      {GRAPH_1_DATA.map(({ label, value }, index) => (
        <ProgressBar
          key={label}
          label={label}
          value={value}
          colorVariant={index}
        />
      ))}
    </div>
  );
}
```
What do You think about something like that? I prefer to have simple ifs instead of ternary operator inside markup. Sometimes it's required but here You have everything nicely separated so You can just define some ifs like i did.